wbooks/src/app/components/authRoute/AuthRoute.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { DEFAULT_AUTH, DEFAULT_NO_AUTH } from '../../utils/Constants';
+import access_token from '../../utils/Constants.js'
+
+class AuthRoute extends React.Component {
+
+
+  render() {
+    const accessToken = localStorage.getItem(access_token);
```

Comment:
Move this to the auth service as a function an import it here, this way you wont need the access_token constant in util/constants. You can define it in the service.